### PR TITLE
feat: Add support for stale bot

### PR DESCRIPTION
**Pull request by lipis**
WDYT?  

Read more on: https://github.com/probot/stale

### CODE REVIEW

**Comment by lipis** [comment]
I'll remove comments if we're about to merge it..

**Comment by gregor** [comment]
I like the idea. Discussed it f2f with @bennyn a couple of weeks ago. I think we should go for it but need to think about the configuration we want to set it to. Can we ensure only stale issues that have no been triaged as valid bugs get closed for example?

**Comment by lipis** [comment]
Of course.. check the line 8.. all we need is to have some labels that are not OK to close..

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale Issue or Pull Request is closed
+daysUntilClose: 7
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - security
+  - "type: Bug"
```
Comment (lipis) [inline .github/stale.yml]: @gregor here..

Comment (gregor) [reply .github/stale.yml]: Thanks for pointing me to it

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# Label to use when marking as stale
+staleLabel: wontfix
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
```
Comment (lipis) [inline .github/stale.yml]: Also words.. :)

**Comment by lipis** [comment]
If we about to merge it.. let's remove the comments and modify maybe a little bit the message and make sure that these are the labels that we want not to be staled..

**Comment by lipis** [comment]
Will merge this one and we can configure it as we go.. for adjustments..

**Comment by lipis** [comment]
Shall we try this little bot?

**Pull request merged**